src/pylhe/__init__.py

```diff
@@ -312,3 +313,35 @@ def visualize(event, outputname):
     tex2pix.Renderer(tex).mkpdf(outputname)
     subprocess.check_call(["pdfcrop", outputname, outputname])
     os.remove("event.dot")
+
+
```

Comment:
I think we can just do a hard rename.. it's a small enough package

Reply:
I was thinking this would give the people we know are using the library at one release grace period. Though as Henry has mentioned in the past, no one really reads warnings. We could probably just tag everyone who has ever opened an Issue or contributed to `pylhe` after this PR is merged and then just announce to them here that we'll be breaking the API in the next release.